core/Core.rdf

```diff
@@ -685,9 +710,9 @@ NL definition: OAGIS and CCO</iof-av:adaptedFrom>
 		<iof-av:adaptedFrom>https://en.wikipedia.org/wiki/Process_(computing) and https://en.wikipedia.org/wiki/Execution_(computing)</iof-av:adaptedFrom>
 		<iof-av:explanatoryNote>1. The inputs and specified outputs of &apos;computing process&apos; are strictly limited to information content entities. 
 2. While it is true that algorithms can result in an action by an agent that concretizes it (e.g. controller changes the pressure of a valve), the intermediate step is still an information content entity (e.g. action specification) that is &apos;concretized&apos; in a separate process that results in the action.</iof-av:explanatoryNote>
-		<iof-av:firstOrderLogicAxiom>ComputingProcess(x) →  ∀y((hasInput(x,y) ∨ hasSpecifiedOutput(x,y)) → InformationContentEntity(y)))</iof-av:firstOrderLogicAxiom>
+		<iof-av:firstOrderLogicAxiom>ComputingProcess(x) → ∀y((hasInput(x,y) ∨ hasSpecifiedOutput(x,y)) → InformationContentEntity(y))</iof-av:firstOrderLogicAxiom>
 		<iof-av:firstOrderLogicDefinition>ComputingProcess(x) ↔ PlannedProcess(x) ∧ ∃y,∃a(Agent(y) ∧ (Algorithm(a) ∨ EncodedAlgorithm(a)) ∧ hasParticipantAtSomeTIme(x,y) ∧ genericallyDependsOnAtSomeTime(a,y) ∧ concretizesAtSomeTime(x,a) ∧ (∃o(ObjectiveSpecification(o) ∧ continuantPartOfAtAllTimes(o,a) ∧ achievesAtSomeTIme(x,o)) ∨ 
```

Comment:
"∃y, ∃a" does not comply with https://www.cs.ox.ac.uk/people/james.worrell/lecture9-2015.pdf as required by https://oagi.atlassian.net/wiki/spaces/IOF/pages/4532142081/IOF+Annotation+Property+Guide+V2.2.

I would suggest "∃y∃a" (or "∃y,a").